docs/data/joy/customization/theme-typography/theme-typography.md

```diff
@@ -2,40 +2,77 @@
 
 <p class="description">Learn about the default theme's typography system and how to customize it.</p>
 
-Joy UI includes a typography system within the theme to help you create consistent text across your application. You can customize the default system or start from scratch depending on your needs.
-
 ## Default system
 
-The default system consists of 13 built-in levels:
+Joy UI's theme includes, by default, a thoughtfully designed 11 levels typography system to help you create consistent UIs.
+Hover over each cell in the table below to see the preview value.
 
-- `body-md` - the baseline typography for the application, used as the default configuration in the [Typography](/joy-ui/react-typography/) and [CssBaseline](/joy-ui/react-css-baseline/) components.
-- `body2` through `body5` - can be used for secondary and tertiary information.
-- The `h1` to `h6` levels follow the semantic HTML headings.
-- The `display1` and `display2` usually appear as taglines for marketing and landing pages.
```

Comment:
Was not feeling like these were adding value ⎯ maybe the "h1 to h6 levels follow the semantic HTML tags" are still useful, but to think about. Open to suggestions!